_articles/upgrade-ubuntu.md

```diff
@@ -73,17 +58,19 @@ Changing this option will require your user password. After selecting your updat
 
 The system will automatically check for updates. You may need to update (and sometimes, restart) before continuing with the upgrade process. If you have to restart after installing updates, launch the updater again as described above.
 
-![Software Updater](/images/upgrade-ubuntu/Step-2.jpg)
+![Software Updater](/images/upgrade-ubuntu/Step-2.png)
+
+![Software Updater](/images/upgrade-ubuntu/Step-2b.png)
 
 #### Install The Upgrade
 
-A message will appear informing you of the availability of the new release. Click **Upgrade** and follow the on-screen instructions.
+A message will appear informing you of the availability of the new release. Click **Yes, Upgrade Now**  or **Upgrade...** depending on the window that you have and follow the on-screen instructions.
 
-![Upgrade Message](/images/upgrade-ubuntu/Step-3.jpg)
+![Upgrade Message](/images/upgrade-ubuntu/Step-3.png)
 
 #### Install The System76 Driver
 
-Once you've upgraded Ubuntu, you'll need to download and install the current System76 Driver. Open the Terminal app (search <u>Terminal</u> from the Ubuntu dash <i class="fl-ubuntu"></i> or press <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>T</kbd>), then enter the following commands:
+Once you've upgraded Ubuntu, you'll need to download and install the current System76 Driver. Open the Terminal app (search <u>Terminal</u> from the Overview <i class="fl-ubuntu"></i> or press <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>T</kbd>), then enter the following commands:
```

Comment:
Because the Overview is labelled "Activities", in some situations I think it may be clearer to refer to it as 
"Activities Overview" rather than just "Overview," because that's the word that people will be looking for.